### DISCUSSION TITLE
Work fine with RTL-SDR Blog V4 Dongle

### DISCUSSION THREAD

**Discussion by Dudestarfan** [KejPi/AbracaDABra#96] [General]
Normally the RTL-SDR Blog V4 dongle is not recognized. But if you swap a file librtlsdr.dll it works fine.
Download the appropriate file: https://github.com/hayguen/librtlsdr/releases
Please unpack and replace the file "librtlsdr.dll" in the AbracaDABra-2.2.4 install directory.
Old RTL-SDR Blog V3 dongles also work.

**Comment by KejPi** [comment]
Be careful, the latest binary was released with this library: https://github.com/old-dab/rtlsdr 
Application will probably crash if you try to change bandwidth. 
I will update rtl-sdr library in next binary release.
